bin/autobahn-router/src/ix_builder.rs

```diff
@@ -145,13 +150,24 @@ impl<T: SwapStepInstructionBuilder> SwapInstructionsBuilder for SwapInstructions
 
         for step in &swap_instructions {
             if auto_create_out || (step.out_mint == sol_mint && auto_wrap_sol) {
-                Self::create_ata(&wallet_pk, &mut setup_instructions, &step.out_mint);
+                let out_is_token_2022 = (step.out_mint != sol_mint)
```

Comment:
could have a list of well-known hardcoded spl_token instead of "step.out_mint != sol_mint"